proto/spiffe/workload/workload.proto

```diff
@@ -38,10 +38,6 @@ message X509SVID {
     // CA certificates belonging to the Trust Domain
     // ASN.1 DER encoded
     bytes bundle = 4;
-
-    // List of trust domains the SVID federates with, which corresponds to
-    // keys in the federated_bundles map in the X509SVIDResponse message.
-    repeated string federates_with = 5;
```

Comment:
if you remove federated_with, you must update following methods:

https://github.com/spiffe/go-spiffe/blob/d8657cb50cae3c05c2d145a559a9681c5376a0c4/workload/proto.go#L70

https://github.com/spiffe/go-spiffe/blob/d8657cb50cae3c05c2d145a559a9681c5376a0c4/spiffetest/workload_api.go#L181

Reply:
Hmm, I'll revert the proto change for now. It needs to be removed because it isn't part of the SPIFFE spec...